python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -78,6 +78,11 @@ class Value extends TObject {
     predicate isBuiltin() {
         this.(ObjectInternal).isBuiltin()
     }
+    
+    /** Retained for backwards compatibility. See Value.isBuiltin() */
+    predicate isC() {
+        this.isBuiltin()
+    }
```

Comment:
Now is the chance, kill all the old things :smile: What I'm thinking is, can we just use `isBuiltin` directly? (then when we're done migrating from Object -> Value, we won't have any `isC()` left)

Reply:
Done.